support deleting vpcs that are non-default (#243)
gruntwork-io/cloud-nuke: README.md

- Deleting all SQS queues in an AWS account
- Deleting all S3 buckets in an AWS account - except for buckets tagged with Key=cloud-nuke-excluded Value=true
- Deleting all default VPCs in an AWS account

> - Deleting all non-default VPCs in an AWS Account - except for default vps which is handled by default-aws

Reviewer: ```suggestion
- Deleting VPCs in an AWS Account (except for default VPCs which is handled by the dedicated `defaults-aws` subcommand)
```
